.editorconfig

```diff
@@ -0,0 +1,12 @@
+root = true
+
+[*]
+charset = utf-8
+end_of_line = lf
+indent_size = 2
+indent_style = space
```

Comment:
So how will this impact if someone uses 4 indent sizes and a tab instead of space?